src/boot/O2/system_malloc.c

```diff
@@ -19,7 +19,7 @@ void SystemArena_Free(void* ptr) {
     __osFree(&gSystemArena, ptr);
 }
 
-void* SystemArena_Calloc(u32 elements, size_t size) {
+void* SystemArena_Calloc(u32 elements, u32 size) {
```

Comment:
A shame

Reply:
What am I missing here, `size_t` used to be `u32` aka `unsigned int`, and with the PR it becomes `unsigned int` so what changed?

Reply:
Actually you are right, this should expand to `unsigned int` like it used to :thinking:

Reply:
Its the fact that `u32` changed to an `unsigned long`, so they now had different types. 
Fwiw it actually matches if you use `size_t` for the first arg and `u32` for the second, and since both instances go unused I just decided to swap the args so they still work.

Reply:
Second argument should match as `size_t` afaict
Also both arguments should be `size_t`, and it would be better to keep the standard nelem,elemsize order
Cf e.g. man calloc https://www.man7.org/linux/man-pages/man3/calloc.3p.html

Reply:
Yeah both work as `size_t`.